crates/primitives/src/bits/macros.rs

```diff
@@ -785,7 +785,8 @@ macro_rules! impl_diesel {
             // #[derive(diesel::AsExpression)]
             impl<Db> ToSql<Nullable<Binary>, Db> for $t
             where
-                for<'c> Db: Backend<BindCollector<'c> = RawBytesBindCollector<Db>>,
+                Db: Backend,
+                [u8]: ToSql<Binary, Db>,
```

Comment:
does

```
Self: ToSql<Binary, Db>,
```

work here?

at least this is what ruint does

https://github.com/recmo/uint/blob/ed6f5fc9f8b1369a28437a7b91cf224b8dbeef53/src/support/diesel.rs#L89C5-L89C29